src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/BreakpointsInference.java

```diff
@@ -343,7 +366,8 @@ void resolveComplications(final SimpleChimera simpleChimera, final byte[] contig
         protected BNDTypeBreakpointsInference(final SimpleChimera simpleChimera,
                                               final byte[] contigSequence,
                                               final SAMSequenceDictionary referenceDictionary) {
-            super(simpleChimera, contigSequence);
+            super(simpleChimera, contigSequence, referenceDictionary);
+            altHaplotypeSequence = new byte[]{};
```

Comment:
But we do have the assembled haplotype.. It's true that it's somewhat redundant but I was thinking it might be nice to have there for consistency with the other variant types. I forget -- do we include any padding in the alt haplotype, or is it really just the inserted sequence in this case?